### ISSUE TITLE
Nova domain NOVA_DOMAIN_NAME is not used in password reset

### ISSUE THREAD

**Issue by mgussekloo** [laravel/nova-issues#3415]
Hi,

Nova's ForgotPasswordController is kind of difficult to overwrite, and it builds the mail message like so: 
`->action(__('Reset Password'), url(config('nova.url').route('nova.password.reset', $token, false)))`

By default, this is configured in the config/nova.php file as: `'url' => env('APP_URL', '/'),`

I'm running Nova on a subdomain with NOVA_DOMAIN_NAME, and I expected the password reset link to point to this subdomain, because otherwise the correct routes are missing. My workaround is to do this in config/nova.php

`
    'domain' => env('NOVA_DOMAIN_NAME', null),
    'url' => env('NOVA_DOMAIN_NAME', '/'),
`

Is this a smart thing to do? Is there a chance the ForgotPasswordController can either be updated, or made to be customized a bit easier?

**Comment by crynobone** [comment]
Can you share repostiory/configuration you used. Kinda hard to follow what the current and expected behaviour based on above description.

**Comment by mgussekloo** [comment]
Thanks for the lightning fast reply!

If route('nova.password.reset') contained the correct domain, I think it would be odd that ForgotPasswordController uses it like so: `url(config('nova.url').route('nova.password.reset')`

This is the beginning of my config:
`
APP_NAME="My App"
APP_ENV=production
APP_KEY=...
APP_DEBUG=false
APP_URL=https://myapp.test
NOVA_DOMAIN_NAME=https://app.myapp.test
`

**Comment by crynobone** [comment]
>  I think it would be odd that ForgotPasswordController uses it like so

That line exists since the first commit. A lot has changed since then. Anyway, I submitted a PR to fix this and waiting to be reviewed.

**Comment by crynobone** [comment]
This has been fixed in v3.27.0: https://nova.laravel.com/releases/3.27.0
